### PR TITLE
Use one request for multiple models

### PR DESCRIPTION
**Pull request by Ladsgroup**
With the new version of ores live, it's much more efficient
Bug: T142360

### CODE REVIEW

File: config.example.wikimedia.yaml [hunk 1]
```diff
@@ -266,9 +266,8 @@ spec: &spec
                         domain: en.wikipedia.org
                       rev_by_bot: false
                     exec:
-                      - uri: 'https://ores.wikimedia.org/v2/scores/enwiki/damaging/{{message.rev_id}}/?precache=true'
-                      - uri: 'https://ores.wikimedia.org/v2/scores/enwiki/reverted/{{message.rev_id}}/?precache=true'
-                      - uri: 'https://ores.wikimedia.org/v2/scores/enwiki/goodfaith/{{message.rev_id}}/?precache=true'
+                      - uri: 'https://ores.wikimedia.org/v2/scores/enwiki/?models=damaging|goodfaith|reverted&
+revids={{message.rev_id}}&precache=true'
```
Comment (Pchelolo) [inline config.example.wikimedia.yaml]: Em, why is there a new line inside the URI? It's not supported.

In general I propose to transform all of them to the better syntax:
```yaml
exec:
    uri: 'https://ores.wikimedia.org/v2/scores/enwiki/
    query:
        models: 'damaging|goodfaith|reverted'
        revids: '{{message.rev_id}}'
        preache: true
```

And also cover the ones that were not touched by this change for consistency.

Comment (d00rman) [inline config.example.wikimedia.yaml]: +1 to what @Pchelolo said, especially the transformation part. It increases readability as well as correctness (encoding, etc)

Comment (Ladsgroup) [inline config.example.wikimedia.yaml]: The puppet change is https://gerrit.wikimedia.org/r/#/c/304125/1. Let me give it a try :)

**Comment by Ladsgroup** [comment]
If this looks okay to you. I make changes to the puppet patch

**Comment by Pchelolo** [comment]
Looks good to me, @Ladsgroup go ahead with puppet. 

The tests are failing because one of them is incorrect, I'll update it in a follow-up PR

**Pull request merged**